### ISSUE TITLE
The pictures of the users days off are not loading properly on the Calendar

### ISSUE THREAD

**Issue by zebraI39** [microsoft/vsts-team-calendar#515]
Hi, when clicking on the Azure DevOps Calendar, the calendar loads but most of the days-offs pictures located on a day somebody is off doesn’t load. Some loads, around 20%.

This happens on Firefox and makes the calendar unusable.

Here is a picture of the problem :
![B28034816f8944c9c816b037a7d458541638555332132212032_20240702-180652-image](https://github.com/microsoft/vsts-team-calendar/assets/123485130/08cea872-7622-4926-8eaa-51c754c2bcb8)

Here is a GIF of the problem :
![Ba03c5e368de1485088f684e1206152db638556077681156497_Animation](https://github.com/microsoft/vsts-team-calendar/assets/123485130/3c2cd742-c142-402b-9149-d59b6e132f32)

Some more information :
- It used to work well. I think it stoped working properly 2 or 3 months ago.
- It happens also to all the other users of the organization who use Calendar.
-It works in Edge and Chrome but not in Firefox.  The problem is that Firefox is the official browser of our organization.
- The result is even worst in a Firefox incognito window with 100% days-off pictures not appearing.

**Comment by PeterLefebvre** [comment]
see something similar with ADS2020 and ADS2022 on premise, in Edge. 
Loading users icons takes a long time and do not always show up in the calendar, only at right. 
Clicking on Today (or next/prev), then users icon start showing up in the calendar

**Comment by zebraI39** [comment]
It is even worst now. No pictures are properly displayed. It makes the calendar totally unusable for me.
Is there another module similar to this one that I can use ?

**Comment by AminTi** [comment]
Hi, I will take a look at it and get back to you

**Comment by danieldorsey3** [comment]
Was there any resolution to this issue? I'm also getting this same issue, in all browsers (chrome, edge).

**Comment by zebraI39** [comment]
> Was there any resolution to this issue? I'm also getting this same issue, in all browsers (chrome, edge).

No, I don't have more information than what is already in this conversation.

**Comment by doormalena** [comment]
Relates to #521 and #523
